server/src/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,22 @@
+import { Review, User } from "../models/index.js";
+
+const getReviewDetail = async (review) => {
+  const user = await review.$relatedQuery("user");
+  const allowedAttributes = ["rating", "description", "id"];
+  let serializedReview = {};
+  for (const attribute of allowedAttributes) {
+    serializedReview[attribute] = review[attribute];
+  }
+  serializedReview.userName = user.name;
+  return serializedReview;
+};
+
+const getReviewsDetails = async (reviews) => {
+  return Promise.all(
+    reviews.map((review) => {
+      return getReviewDetail(review);
+    })
+  );
+};
+
```

Comment:
Defining these functions on a ReviewSerializer class or as an independent function is a design decision that you once again should have as a team. On principle Im actually okay with this as its own standalone function, but while in group projects it's veering away from norms in a way that could get confusing to the rest of the group. 

Have you decided you will follow this pattern for every serializer? 
Will everyone understand it? 
If so then its technically okay....

But also: you don't use `getReviewDetail` anywhere. I understand you may use it somewhere else in the future, but if being unused in a PR, it will be rejected

Reply:
Yep -refactored to a class.  The getReviewDetail is getting used by the getReviewDetail

Reply:
Adjusting the names of these methods can help provide clarity around the purpose of each of these methods, and distinguish them from one another. Like `getReviewDetails` vs `getReviewCollectionDetails`

Reply:
Changed name to getReviewCollectionDetails